src/frontend/shortlist/src/api/shortlist.js

```diff
@@ -104,15 +109,19 @@ export class temporarySignup {
           .then((result) => this.successCb(result))
```

Comment:
Maybe delete these commented out codes, unless they might be a reference for future contributors?